project.clj

```diff
@@ -0,0 +1,8 @@
+(defproject ecs-cost-estimator "0.1.0-SNAPSHOT"
+  :description "Parse ECS terraform file and calculate estimated min and max cost per month."
+  :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
+            :url  "https://www.eclipse.org/legal/epl-2.0/"}
+  :dependencies [[org.clojure/clojure "1.10.0"]
+                 [camel-snake-kebab "0.4.0"]
+                 [com.bertramlabs.plugins/hcl4j "0.2.0"]]
+  :repl-options {:init-ns ecs-cost-estimator.core})
```

Comment:
minor: you may want to append `:main ecs-cost-estimator.core` so that usage command can be simplified to
```sh
lein run /path/to/a/terraform.tf
```